website/docs/usage/training.md

```diff
@@ -355,6 +356,58 @@ that reference this variable.
 
 </Infobox>
 
+## Preparing Training Data {#training-data}
+
+Training data for NLP projects comes in many different formats. For some common
+formats such as CoNLL, spaCy provides [converters](/api/cli#convert) you can use
+from the command line. In other cases you'll have to prepare the training data
+yourself.
+
+When converting training data for use in spaCy, the main thing is to create Doc
```

Comment:
```suggestion
When converting training data for use in spaCy, the main thing is to create [`Doc`](/api/doc)
```
(this might require a new run of `prettier`)